### ISSUE TITLE
Thermal metals turn in to Railcraft metals in crafting table / grid

### ISSUE THREAD

**Issue by GrumpyChunks** [FTBTeam/FTB-Revelation-2#226]
* **Modpack Version**: 3.0.1

* **Issue**: Assuming this isn't by design (please close if so), but when changing the form of thermal metal in a crafting table/grid (i.e. nuggets <-> ingots <-> blocks), the metal will assume that of railcraft instead of thermal expansion. e.g. place a block of thermal expansion copper into a crafting table, the output is 9 ingots of railcraft copper. No matter what metal (or form the metal is in), the result always seems to be railcraft metals. Given the ores in the world are Thermal ores, it seems sensible that Thermal metals are the default / higher priority.

* **Link to Log or Crash File Paste**: No crash

* **Is it Repeatable?**: yes, 100% of the time in single player

* **Mod/s Affected**: Thermal Expansion / Railcraft / (Unidict?)

* **Known Fix**: I've added three lines to scripts/01_unification.zs, as follows:

```
	# Railcraft OreDict Fixes
recipes.remove(<railcraft:ingot:*>);
recipes.remove(<railcraft:metal:*>);
recipes.remove(<railcraft:nugget:*>);
recipes.addShaped(<railcraft:metal:8>,
	[[<railcraft:ingot:8>,<railcraft:ingot:8>,<railcraft:ingot:8>],
	[<railcraft:ingot:8>,<railcraft:ingot:8>,<railcraft:ingot:8>],
	[<railcraft:ingot:8>,<railcraft:ingot:8>,<railcraft:ingot:8>]]
);
recipes.addShaped(<railcraft:ingot:8>,
	[[<railcraft:nugget:8>,<railcraft:nugget:8>,<railcraft:nugget:8>],
	[<railcraft:nugget:8>,<railcraft:nugget:8>,<railcraft:nugget:8>],
	[<railcraft:nugget:8>,<railcraft:nugget:8>,<railcraft:nugget:8>]]
);
recipes.addShapeless(<railcraft:ingot:8>*9,
	[<railcraft:metal:8>]
);
recipes.addShapeless(<railcraft:nugget:8>*9,
	[<railcraft:ingot:8>]
);
```

Not sure if the change I'm using might have caused other side effects (possibly later in the game if/when I start using railcraft), but it's solved the unification issue for me for now at least. It's not a big issue, but it's really annoying having different types of copper ingots (for example) in my storage.

Edit: Have found an issue with adding the above, it prevents Zinc nuggets (not a thermal metal) from being crafted into Zinc ingots in the crafting table, though a smeltery can be used to melt them down and then cast into railcraft zinc ingots.

Edit2: Updated the script above, this resolves the issue with Railcraft Zinc

**Comment by DeeDoubleYous** [comment]
I can confirm that I got this as well. It's not game breaking, but still a quality of life frustration as it introduces an ingot that otherwise would not be possible to get taking up more storage.

**Comment by DrGirlfriend** [comment]
Same here, is there a quick fix in the config files for this?

**Comment by fguillod** [comment]
Also got this with copper and tin blocks that turn into Railcraft ingots.

**Comment by tfox83** [comment]
This has been fixed in 3.1.0
